src/engines/witcher/lua/bindings.h

```diff
@@ -424,6 +424,667 @@ class LuaBindings {
+	uint32_t _logTime;
+	uint32_t _forceLogAllTimes;
+	float    _floatingInfoMaxCameraDist;
+	float    _floatingInfoMinPlayerDist;
+	uint32_t _floatingInfoInFrontOfPlayerOnly;
+	uint32_t _debugScripts;
+	uint32_t _debugMusicians;
+	uint32_t _enableMusiciansCrossfadeOffset;
+	uint32_t _enableVMProfiler;
+	uint32_t _logExecutedCommands;
+	float    _mapRevealRange;
+	float    _mapRevalInterval;
+	uint32_t _useOldPanels;
+	uint32_t _quickExit;
+	uint32_t _subtitles;
+	uint32_t _resManUsesWindowsMemMapping;
+	uint32_t _dumpAlchemyDebugInfo;
+	uint32_t _dialogCameraDebug;
+	uint32_t _maxUserMarkersOnMap;
+	uint32_t _bumpFloatingBubbles;
+	uint32_t _physicsMaxRDCollisions;
+	float    _physicsLowFPSThreshold;
+	uint32_t _dontPlayCutscenes;
+	uint32_t _dontPlayMovies;
+	uint32_t _bribeAmountVariationMaxPercentage;
+	float    _maxTooltipVisibilityDistance;
+	float    _cameraJumpDist;
+	float    _getSurfaceMatDelay;
+	uint32_t _dblClickMaxMouseMoveSquared;
+	uint32_t _dblClickOverride;
+	float    _combatModeBaseDist;
+	uint32_t _dumpFistfightDebug;
+	float    _groupStylePushDistFactor;
+	float    _groupStyleHitDistFactor;
+	uint32_t _physicsDebugMode;
+	float    _physicsWindAffectRange;
+	uint32_t _physicsSimulationAlwaysOn;
+	uint32_t _physicsUseReducedAABB;
+	uint32_t _combatModeTime;
+	float    _tNTMinDistToWalk;
+	float    _tNTFadeTime;
+	float    _tNTPreDialogShotTime;
+	float    _combatModeGenericDist;
+	float    _improvisedAttackImpact;
+	uint32_t _minigamesDebug;
+	uint32_t _minigamesDebugCamera;
+	float    _minigamesSceneAmbient;
+	uint32_t _minigamesNonBSPLighting;
+	uint32_t _cursorSignalizingSequence;
+	uint32_t _curSignSeqMaxGameDifficulty;
+	uint32_t _resetAmbientAfterCutscene;
+	uint32_t _drawAttackRange;
+	uint32_t _doublePressTimeMs;
+	uint32_t _showTutorials;
+	float    _pVSCellZMin;
+	float    _pVSCellZMax;
+	float    _pVSCellDetectionDistance;
+	float    _pVSCellDetectionStep;
+	float    _simpleSkeletonDist;
+	uint32_t _gameDifficultySetting;
+	uint32_t _reportDialogAndCutsceneScripts;
+	uint32_t _enableManualActorOrientation;
+	float    _personalSpaceMultiplier;
+	uint32_t _crazyLittleThing;
+	float    _gammaSetting;
+	uint32_t _nearbyDoorUpdateTime;
+	float    _nearbyDoorPSMultiplier;
+	float    _nearbyDoorDist;
+	uint32_t _patchQuests;
+	uint32_t _pauseStateAtomic;
+	uint32_t _dumpMapTrackingMessages;
+	uint32_t _validateStringDuplicatesOnSave;
+	float    _videoMemoryDivider;
+	uint32_t _oTSCameraSide;
+	uint32_t _unpackSaveBeforeLoad;
+	uint32_t _profileAreaLoading;
+	uint32_t _enableLiveContentPanels;
+	uint32_t _isLiveModLoaded;
+	uint32_t _showFloatingLookatInfo;
+	float    _lookatPCChance;
+	uint32_t _showFloatingEmotionalState;
+	uint32_t _dontYouEverTouchMyGamma;
+	uint32_t _skipCutscenesOneByOne;
+	uint32_t _speedTreeRenderBranches;
+	uint32_t _speedTreeRenderFronds;
+	uint32_t _speedTreeRenderLeafCards;
+	uint32_t _speedTreeRenderBillboards;
+	uint32_t _dumpLoadingTextures;
+
+	CAuroraSettingsData();
```

Comment:
This won't compile, since you add the constructor definiton in a later commit.

But also, can you do C++11-style inline initializations here instead? I.e.
`float _attackLockDist { 6.0f };`